plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/index.tsx

```diff
@@ -221,7 +221,9 @@ const ProductCollectionInspectorControls = (
 					) }
 				</ToolsPanel>
 			) : null }
-			<ProductCollectionFeedbackPrompt />
+			<PanelBody>
```

Comment:
While working on another, unrelated issue regarding tracks I realised we should not be displaying feedback option if user doesn't allow for tracking (if tracking is disabled, feedback won't be sent anyway).

This is the way to check if tracking is enabled:

```
window.wcTracks?.isEnabled
```

and here's example:

https://github.com/woocommerce/woocommerce/blob/3f6b19126e26235ba6df5688632e96f8be6a08f5/packages/js/product-editor/src/hooks/use-feedback-bar/use-feedback-bar.ts#L30

Please add it before merging.